packages/atomic/src/utils/utils.ts

```diff
@@ -38,3 +38,7 @@ export function sanitize(string: string) {
   const reg = /[&<>"'/]/gi;
   return string.replace(reg, (match) => map[match]);
 }
+
+export function parseXML(string: string) {
+  return new window.DOMParser().parseFromString(string, 'text/xml');
```

Comment:
I believe `DOMParser` will throw if the string contains invalid XML. If so, we should catch and recover. Otherwise, one badly indexed result might crash the UI, or cause it to work incorrectly.

I think it would also be useful to also move the `Array.from` step into the util function.

Reply:
Array from is used after having fetched the list of parent xml elements, it's not really of any use in the util itself.